src/pages/Inventory.jsx

```diff
@@ -15,7 +14,8 @@ import Select from "../components/Select";
 import "./Inventory.css";
 import { BacktraceClient } from "@backtrace-labs/react";
 
-const Inventory = () => {
+const Inventory = ({data}) => {
```

Comment:
Perhaps we should have this be `data = InventoryData` with the original import so we default to the current behavior? Would make it require no changes to the existing tests, too.

Reply:
I introduced a second route for an inventory with different set of data. In future will probably add lazy loading feature there and I think it will be easier to test with a prop instead of dealing with routes to define which data should be used. With next iteration I can change prop into `longData: boolean` and deal importing data in only one place.